### ISSUE TITLE
Search for address -- auto-fill is hidden on mobile phone

### ISSUE THREAD

**Issue by shodge17** [FREAC/COM#52]
This can only be seen on an actual phone.  When the keyboard pops up it makes the screen much smaller so only the top part of the suggested addresses can be seen. 

Perhaps the simplest solution would be to move the address search above the "locate me" button.  That might just be enough to show at least one or two suggestions.

**Comment by clintonlunn** [comment]
@shodge17  Do you mean this address search? 

![image](https://user-images.githubusercontent.com/24685932/58925122-5bc14800-8714-11e9-9b33-d1daefb29b29.png)

Or do you mean the provider search? 

![image](https://user-images.githubusercontent.com/24685932/58925179-93c88b00-8714-11e9-9782-1e07514133dd.png)

**Comment by shodge17** [comment]
The very top image - after clicking the magnifying glass

**Comment by clintonlunn** [comment]
@shodge17 Okay, thanks for the clarification. I can rearrange these controls today.

**Comment by clintonlunn** [comment]
I was able do to this, and ran into a little bit of a problem:
![image](https://user-images.githubusercontent.com/24685932/59076438-09af2c80-88a4-11e9-9798-361303222e87.png)

As an alternate approach, leaflet has a method to detect if on mobile browser. This would give us more room to view the address autocomplete. 

Code snippit to detect mobile browser:
```
// if mobile browser true
 if (L.Browser.mobile) {
    // Do something
 }
```

**Comment by clintonlunn** [comment]
Also, since we can detect if on browser, _maybe_ we can also detect when the keyboard is raised. I have researched this a little bit, but I haven't yet found this event trigger. 

Maybe it would be a useful case that when the keyboard is raised:
- The provider search bar is moved to floating on the top of the map
- Legend panel reverts to display: none

 Then, once the map keyboard is lowered:
- Legend display re-enabled
- Provider search is moved back into legend. 

Further from that, I think I think having the search bar always at the top of the map would put it out of the way of the raised keyboard, and give more room for results to display. I know we had gone away from this solution before, but I figured it was at least partially because of all of the funky rules and formatting that needed to be ironed out with bootstrap classes. I was wondering your thoughts on this as possible future functionality.
